site/content/api/Credentials-API.md

```diff
@@ -0,0 +1,323 @@
+**Response Message Format**
+
+A response to an *update credentials* request contains the [Standard Response Properties]({{< relref "#standard-response-properties" >}}).
+
+The response message's body is empty. 
+
+The response message's *status* property may contain the following codes:
+
+| Code  | Description |
+| :---- | :---------- |
+| *204* | No Content, the credentials have been updated successfully. |
+| *404* | Not Found, there are no credentials registered matching the critera from the payload. |
+
+For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
+
+## Remove Credentials
+
+Clients use this command to *remove* credentials registered for a device. Once the credentials are removed, the device may no longer be able to authenticate with protocol adapters using the authentication mechanism that the removed credentials corresponded to.
+
+**Message Flow**
+
+*TODO* add sequence diagram
+
+**Request Message Format**
+
+The following table provides an overview of the properties a client needs to set on a *remove credentials* message in addition to the [Standard Request Properties]({{< relref "#standard-request-properties" >}}).
+
+| Name             | Mandatory | Location                 | Type           | Description |
+| :--------------- | :-------: | :----------------------- | :------------- | :---------- |
+| *subject*        | yes       | *properties*             | UTF-8 *string* | MUST contain the value `remove`. |
+
+The body of the message MUST consist of a single *AMQP Value* section containing a UTF-8 encoded string representation of a single JSON object as follows:
+
+~~~json
+{
+  "device-id": "${device_id}",
+  "type": "${credential_type}",
+  "auth-id": "${authentication_identifier}"
+}
+~~~
+
+The `device-id` property MUST contain the ID of the device from which the credentials should be removed.
+The `tpye` property indicates the type of credentials to remove. If set to `*` then all credentials of the device are removed (`auth-id` is ignored), otherwise only the credentials matching the `type` and `auth-id` are removed.
+The `auth-id` property is optional and if omitted, then all credentials of the specified `type` are removed.
+
+**Response Message Format**
+
+A response to a *remove credentials* request contains the [Standard Response Properties]({{< relref "#standard-response-properties" >}}).
+
+The response message's body is empty.
+
+The response message's *status* property may contain the following codes:
+
+| Code  | Description |
+| :---- | :---------- |
+| *204* | No Content, the credentials have been removed successfully. |
+| *404* | Not Found, there are no credentials registered matching the critera given in the request payload. |
+
+For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
+
+# Standard Message Properties
```

Comment:
I like the description of standard properties for all requests (making the reading significantly better).
I think we should do the same in the other API descriptions as well.
WDYT?